### ISSUE TITLE
Adding log-transformed inputs

### ISSUE THREAD

**Issue by dlinzner-bcs** [experimental-design/bofire#349]
We frequently encounter variables that are better modeled in the logarithmic space. It would be a cool feature to have bofire to model this directly.

@niklaswulkow please add/correct ;)

**Comment by jduerholt** [comment]
Good point, I have it already for a longer time on my bucket list, from the `botorch` side, this is already prepared: 
https://github.com/pytorch/botorch/blob/68fbb68d4a78efd2b803071d883081b2c45a70ae/botorch/models/transforms/input.py#L878
The question is how to set it up in BoFire. I would vote for setting up the domain still in the original non-transformed space and just tell the surrogate which input to transform, as it is currently also done for the input scalers fors continuous inputs. This needs to be made more flexible. 

What do you think?

cc: @bertiqwerty: maybe we can discuss this also tmr in our meeting

**Comment by jduerholt** [comment]
Furthermore, the question would be if the log transformed inputs should then also undergo normalization/standardization and how to set this up in a flexible way.

**Comment by jduerholt** [comment]
https://ax.dev/docs/models.html#transforms This describes how it is done in Ax, which differs a bit from our current approach.

**Comment by psosnin** [comment]
Hello, I am new to bofire and my use case is hyper-parameter tuning of machine learning models. This feature would be quite useful in my setting as many hyper-parameters are well modelled in logarithmic space. I was wondering what the status of this issue is and if there are plans to add this feature?

**Comment by jduerholt** [comment]
Hi Philpp, 

currently nobody is working on this, still we want to have it in at some point. But you can of course manually transform it to log space when setting up the problem and manually back transform when you have generated the candidates.

I hope this helps!

Best,

Johannes
